adafruit_turtle.py

```diff
@@ -1,6 +1,7 @@
 # SPDX-FileCopyrightText: 2006-2010 Gregor Lingl for Adafruit Industries
 # SPDX-FileCopyrightText: 2019 LadyAda for Adafruit Industries
 # SPDX-FileCopyrightText: 2021 Dave Astels for Adafruit Industries
+# SPDX-FileCopyrightText: 2022 Carsten Thue-Bludworth for Adafruit Industries
```

Comment:
I think the intention is to only use `for Adafruit Industiries` if you were paid by them for the development work.

Reply:
Haha, ok I understand (I wish I _was_ working for Adafruit!). I added it because I was trying to follow all of the proper protocols for contributing according to [this](https://learn.adafruit.com/contribute-to-circuitpython-with-git-and-github/add-author-and-license-information) guide.